app/styles/components/_learner-group-overview.scss

```diff
@@ -0,0 +1,13 @@
+.toggle-onoff {
```

Comment:
This is too generic and will end up hitting this box when we place it other places where it is not aligned like this.  I think we probably need to create an alignment class that is more generic and maybe put it in _detailView.